nativelib/src/main/scala/scala/scalanative/native/time.scala

```diff
@@ -0,0 +1,405 @@
+package scala.scalanative
+package native
+
+@extern
+object time {
+
+  // Types
+  /** Arithmetic (until C11) Real (since C11) type capable of representing times.
+   * Although not defined by the C standard, this is almost always an integral value holding the number of seconds
+   * (not counting leap seconds) since 00:00, Jan 1 1970 UTC, corresponding to POSIX time
+   *
+   */
```

Comment:
We don't leave comments in the bindings for C headers.

Reply:
Please remove all comments in this file.

Reply:
:''(
Taking them out but I don't understand why, wouldn't that help with scaladoc tooltips ala intellisense?

Reply:
This is a good point. Lets leave them out for now and open a new issue for adding docs to all the bindings we ship. We need to agree upon what's the detail of documentation we want to include there. Also we should use a consistent source for the docs everywhere.